corehq/messaging/scheduling/util.py

```diff
@@ -0,0 +1,8 @@
+from datetime import datetime
+
+
+def utcnow():
+    """
+    Defined here to do patching in tests.
```

Comment:
yeah, datetime.utcnow() was used in a lot of different files though so it seemed easier for patching to centralize it this way